### PR TITLE
feat: add "Migrated" date tag to instance history

### LINKED ISSUE #18576: I can see the date of the last successful migration for a process instance
Related to https://github.com/camunda/zeebe/issues/17704

### User Story

```Gherkin
Scenario: As an Operate user
    Given I am in process instance details view
    Then I see a tag with the date of the last successful migration operation
```
### Designs
https://www.figma.com/design/tnqak7ht4Hb4VCGygvLNFU/Version-Migration-HFW?node-id=1970%3A164429&t=EpWVp0Rz3Aw2Hajq-1

### Acceptance Criteria

In case a process instance has successfully been migrated:
- [x] A tag is added to the instance history (next to the process element)
- [x] The tag shows the date of the last successful migration

In case there is no successful migrate operation
- [x] No tag is shown

### Definition of Ready - Checklist

- [x] The issue has a meaningful title, description, and testable acceptance criteria
- [ ] If documentation needs to be updated, an issue is created in the [camunda-docs](https://github.com/camunda/camunda-docs) repo, and the issue is added to our Operate project board.
- [ ] If HELM charts need to be updated, an issue is created in the [camunda-platform-heml](https://github.com/camunda/camunda-platform-helm) repo, and the issue is added to our Operate project board.
- [ ] Potential Security risks have been considered and added to the product-hub issue risk assessment
- [ ] Cross-team dependencies have been considered

Optional:
- [ ] Design input has been collected by the assignee
### :point_right: Handover Dev to QA 

- Handy resources: 
     BPMN/DMN models, plugins, scripts, REST API endpoints + example payload, etc : 
     
- Example projects:

- Versions to validate:
> Docker file : in case that it needed to be tested via docker share the version contain the fixed along with version of other services . 

- Release version ( in which version this fixed/feature will be released)

### PR DESCRIPTION
**Pull request by pedesen**
## Description

This PR adds the "Migrated" date tag to the instances history along with some refactorings a feature flag and mock response for the process instance request.

## Related issues

related to #17704
closes #18576

### CODE REVIEW

File: operate/client/e2e-playwright/pages/Processes/Processes.ts [hunk 1]
```diff
@@ -6,48 +6,23 @@
...
-import {Page, Locator, expect} from '@playwright/test';
+import {Page, Locator} from '@playwright/test';
 import {convertToQueryString} from '../../utils/convertToQueryString';
 import {Paths} from 'modules/Routes';
 import {DeleteResourceModal} from '../components/DeleteResourceModal';
 import MigrationModal from '../components/MigrationModal';
 import MoveModificationModal from '../components/MoveModificationModal';
 import {Diagram} from '../components/Diagram';
-
-type OptionalFilter =
```
Comment (pedesen) [inline operate/client/e2e-playwright/pages/Processes/Processes.ts:17]: I moved all filters panel related things to a new FiltersPanel class. I realized that the Processes class grew very big and it was hard to find things.

File: operate/client/e2e-playwright/pages/Processes/FiltersPanel.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    });
+
+    this.variableNameFilter = this.panel.getByRole('textbox', {
+      name: /name/i,
+    });
+
+    this.variableValueFilter = this.panel.getByRole('textbox', {
+      name: /value/i,
+    });
+
```
Comment (vitorwtavares) [inline operate/client/e2e-playwright/pages/Processes/FiltersPanel.ts:105]: Could we add a selector for the multiple mode toggle on the variable filter too?

**Review by vitorwtavares — APPROVED** [review]

File: operate/client/e2e-playwright/pages/Processes/FiltersPanel.ts (hunk 2 above)
Comment (pedesen) [reply operate/client/e2e-playwright/pages/Processes/FiltersPanel.ts:105]: Good idea. I'll add it.

**Comment by pedesen** [comment]
@gastonpillet01 Please use this image to test locally: `registry.camunda.cloud/team-operate/camunda-operate:branch-fe-18576-migration-completed-date-tag`

File: operate/client/e2e-playwright/tests/processInstanceMigration.spec.ts [hunk 3]
```diff
@@ -39,23 +45,18 @@ test.beforeAll(async ({request}) => {
     .toHaveProperty('total', 10);
 });
 
-test.beforeEach(async ({processesPage}) => {
-  await processesPage.navigateToProcesses({searchParams: {active: 'true'}});
-});
-
-test.describe('Process Instance Migration @roundtrip', () => {
+test.describe.serial('Process Instance Migration @roundtrip', () => {
```
Comment (pedesen) [inline operate/client/e2e-playwright/tests/processInstanceMigration.spec.ts:48]: Here I used `serial` which now runs the 2 tests in a sequence.

**Comment by pedesen** [comment]
Hey @vitorwtavares Could you please review the last 2 PRs? In the meantime the backend API changes were merged and I added an e2e test. Thanks!

**Review by vitorwtavares — APPROVED** [review]

**Review by gastonpillet01 — APPROVED** [review]
Migrated tag is displayed correctly after migration is applied.

**Comment by pedesen** [comment]
@Szik Please test locally using this docker image name: `registry.camunda.cloud/team-operate/camunda-operate:branch-fe-18576-migration-completed-date-tag`

**Review by Szik — APPROVED** [review]

**Pull request merged**